cloud/docker/docker.py

```diff
@@ -97,9 +97,12 @@
       - You can specify a different logging driver for the container than for the daemon.
```

Comment:
This description is a little verbose - shouldn't it just link to the [Docker reference](https://docs.docker.com/reference/logging/overview/) page instead?